### PR TITLE
[kesch] Kesch production

### PR DESCRIPTION
**Pull request by lucamar**
The pull request is merging the production lists of branches `rh7.5-17.06` and `rh7.5-18.12` in production for the updated system.

### CODE REVIEW

**Comment by lucamar** [comment]
Note that Cray builds will fail in `TestingEB` since the following options should be used (see the production list): `--module-naming-scheme=EasyBuildMNS --optarch=haswell`.

File: easybuild/easyconfigs/c/CrayCCE/CrayCCE-17.06.eb [hunk 1]
```diff
@@ -9,11 +10,12 @@ description = """Toolchain using Cray compiler wrapper, using PrgEnv-cray module
...
 
 dependencies = [
-    # PrgEnv version is not pinned, as Cray recommends to use the latest (default) version
-    ('PrgEnv-cray', EXTERNAL_MODULE),
+    # PrgEnv version is pinned to avoid conflicts, although Cray recommends to use the latest (default) version
+    ('PrgEnv-cray/1.0.2', EXTERNAL_MODULE),
     ('cce/8.6.0', EXTERNAL_MODULE),
     ('cray-libsci/17.06.1', EXTERNAL_MODULE),
     ('mvapich2_cce/2.2rc1.0.3_cuda80', EXTERNAL_MODULE),
+    ('slurm/17.11', EXTERNAL_MODULE)
```
Comment (kraushm) [inline easybuild/easyconfigs/c/CrayCCE/CrayCCE-17.06.eb]:
```suggestion
```

File: easybuild/easyconfigs/c/CrayCCE/CrayCCE-18.12.eb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+dependencies = [
+    # PrgEnv version is pinned to avoid conflicts, although Cray recommends to use the latest (default) version
+    ('PrgEnv-cray/1.0.4', EXTERNAL_MODULE),
+    ('cce/8.7.7', EXTERNAL_MODULE),
+    ('cray-libsci/18.07.1', EXTERNAL_MODULE),
+    ('cray-mvapich2/2.3', EXTERNAL_MODULE),
+    ('slurm/17.11', EXTERNAL_MODULE)
```
Comment (kraushm) [inline easybuild/easyconfigs/c/CrayCCE/CrayCCE-18.12.eb]:
```suggestion
```

File: easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.3-GCC-5.4.0-2.26-cuda-9.2.eb [hunk 3]
```diff
@@ -0,0 +1,42 @@
+name = 'MVAPICH2'
+version = '2.3'
+cudaversion = '9.2'
+versionsuffix = '-cuda-%s' % cudaversion
+
+homepage = 'http://mvapich.cse.ohio-state.edu/overview/mvapich2/'
+description = """MVAPICH2/2.2 (includes MPICH-3.1.4)"""
```
Comment (kraushm) [inline easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.3-GCC-5.4.0-2.26-cuda-9.2.eb]:
```suggestion
description = """MVAPICH2/2.3 (includes MPICH-3.1.4)"""
```

File: easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.3-PGI-18.10-5.4.0-2.26-cuda-9.2.eb [hunk 4]
```diff
@@ -0,0 +1,43 @@
+# contributed by Luca Marsella (CSCS)
+name = 'MVAPICH2'
+version = '2.3'
+cudaversion = '9.2'
+versionsuffix = '-cuda-%s' % cudaversion
+
+homepage = 'http://mvapich.cse.ohio-state.edu/overview/mvapich2/'
+description = """MVAPICH2/2.2 (includes MPICH-3.1.4)"""
```
Comment (kraushm) [inline easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.3-PGI-18.10-5.4.0-2.26-cuda-9.2.eb]:
```suggestion
description = """MVAPICH2/2.3 (includes MPICH-3.1.4)"""
```

File: easybuild/easyconfigs/s/Szip/Szip-2.1.1-PGI-18.5-GCC-5.4.0-2.26.eb [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+toolchain = {'name': 'PGI', 'version': '18.5-GCC-5.4.0-2.26'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.hdfgroup.org/ftp/lib-external/%(namelower)s/%(version)s/src']
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = '--with-pic'
+
+
```
Comment (kraushm) [inline easybuild/easyconfigs/s/Szip/Szip-2.1.1-PGI-18.5-GCC-5.4.0-2.26.eb]:
```suggestion
```

**Pull request merged**